examples/default/Makefile

```diff
@@ -34,6 +34,12 @@ USEMODULE += ps
 # include and auto-initialize all available sensors
 USEMODULE += saul_default
 
+/* TODO: drop this before merging */
+USEMODULE += shell_lock
+USEMODULE += shell_lock_auto_locking
+CFLAGS += -DSHELL_LOCK_PASSWORD=\"pass\"
+CFLAGS += -DSHELL_LOCK_AUTO_LOCK_TIMEOUT_MS=7000
```

Comment:
Just make a new application in `tests/`, this can also include a python test script to automatically test the functionality.